quickstarts/minikube/index.md

```diff
@@ -34,33 +35,26 @@ After that we feed Strimzi with a simple **Custom Resource**, which will than gi
 
 ```shell
 # Apply the `Kafka` Cluster CR file
-kubectl apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{{site.data.releases.operator[0].version}}/examples/kafka/kafka-persistent-single.yaml -n kafka
+kubectl -n kafka apply -f https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/{{site.data.releases.operator[0].version}}/examples/kafka/kafka-persistent-single.yaml 
```

Comment:
Why did you moved the namespace? It is the thing which the users will most probably want to change, so having it at the end of the commands makes most sense.

Reply:
It was inconsistent before (some commands had it at the start), but I've now put it back at the end for almost all of them. The exception is kubectl run where the console producer/consumer command line precludes putting it at the end. Wdty?